actors/migration/nv15/top.go

```diff
@@ -57,6 +53,28 @@ type MigrationCache interface {
 	Load(key string, loadFunc func() (cid.Cid, error)) (cid.Cid, error)
 }
 
+func ActorHeadKey(addr address.Address, head cid.Cid) string {
+	return addr.String() + "-h-" + head.String()
+}
+
+func SectorsAmtKey(sectorsAmt cid.Cid) string {
+	return "s-" + sectorsAmt.String()
+}
+
+func DeadlineKey(dl cid.Cid) string {
+	return "d-" + dl.String()
+}
```

Comment:
Can we use `StringOfBase` here? Just to be clear.

Reply:
Looks like that would mean that the Key methods would have to return an error (or do something else about the error), which isn't great...

Reply:
Panic? That can only happen if the encoding isn't supported.

Basically, `dl.String()` isn't guaranteed to a _specific_ base.

Reply:
Sure, I was too scared to do that :P